src/NexusMods.App.UI/Pages/MyLoadouts/GameLoadoutsSectionEntry/GameLoadoutsSectionEntryViewModel.cs

```diff
@@ -1,15 +1,130 @@
 using System.Collections.ObjectModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Avalonia.Threading;
+using DynamicData;
+using DynamicData.Binding;
 using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Games;
+using NexusMods.Abstractions.Loadouts;
 using NexusMods.App.UI.Controls.LoadoutCard;
+using NexusMods.App.UI.Pages.LoadoutGrid;
+using NexusMods.App.UI.Resources;
+using NexusMods.App.UI.Windows;
+using NexusMods.App.UI.WorkspaceSystem;
+using NexusMods.MnemonicDB.Abstractions;
+using ReactiveUI;
 
 namespace NexusMods.App.UI.Pages.MyLoadouts.GameLoadoutsSectionEntry;
 
 public class GameLoadoutsSectionEntryViewModel : AViewModel<IGameLoadoutsSectionEntryViewModel>, IGameLoadoutsSectionEntryViewModel
 {
-    public GameLoadoutsSectionEntryViewModel(GameInstallation gameInstallation)
+    private readonly CompositeDisposable _compositeDisposable;
+    private readonly GameInstallation _gameInstallation;
+    private readonly IWindowManager _windowManager;
+    private readonly ReadOnlyObservableCollection<IViewModelInterface> _loadoutCardViewModels;
+    private readonly ReadOnlyObservableCollection<IViewModelInterface> _cardViewModels;
+    private readonly SourceList<IViewModelInterface> _cardViewModelsSourceList = new();
+    public string HeadingText { get; }
+    public ReadOnlyObservableCollection<IViewModelInterface> CardViewModels => _cardViewModels;
+
+
+    public GameLoadoutsSectionEntryViewModel(GameInstallation gameInstallation, IConnection conn, IServiceProvider serviceProvider, IWindowManager windowManager)
     {
-        HeadingText = gameInstallation.Game.Name + " Loadouts";
+        _compositeDisposable = new CompositeDisposable();
```

Comment:
Dispose in `this.WhenActivated`

Reply:
Since this is not actually used as a VM for a specific view, but rather it is just used as source of bindings for parts of MyLoadoutsView, I don't think `WhenActivated` is actually called, that is why it is disposed in `Dispose()` instead.

Reply:
Odd

Reply:
I ended up using DisposeMany() rather than the `activator.Activate()` and `Activator.Deactivate()` , which felt like a hack.